spec/lib/audit/moab_to_catalog_spec.rb

```diff
@@ -191,7 +191,8 @@
 
       storage_dir_a_seed_result_lists = described_class.seed_catalog_for_dir(storage_dir_a)
       expect(storage_dir_a_seed_result_lists.count).to eq 1
-      expected_result_msg = 'db update failed: #<ActiveRecord::RecordInvalid: Validation failed: Preserved object has already been taken>'
+      expected_result_msg = 'db update failed: #<ActiveRecord::RecordNotSaved: Failed to remove the existing associated complete_moab. ' \
+                            'The record failed to save after its foreign key was set to nil.>'
```

Comment:
@jmartin-sul this is the new spec from your PR #1954;  I assume this tweak is ok with you.